### ISSUE TITLE
Build failure: lldap 

### ISSUE THREAD

**Issue by Scandiravian** [NixOS/nixpkgs#268571]
### Steps To Reproduce

1. build lldap from unstable with Rust 1.73.0

### Build log

```
Error: wasm32-unknown-unknown target not found in sysroot: "/nix/store/gz7llp706g0apmsiswwisbpxqdmy0sfk-rustc-1.73.0"

Used rustc from the following path: "/nix/store/gz7llp706g0apmsiswwisbpxqdmy0sfk-rustc-1.73.0/bin/rustc"
It looks like Rustup is not being used. For non-Rustup setups, the wasm32-unknown-unknown target needs to be installed manually. See https://rustwasm.github.io/wasm-pack/book/prerequisites/non-rustup-setups.html on how to do this.

Caused by: wasm32-unknown-unknown target not found in sysroot: "/nix/store/gz7llp706g0apmsiswwisbpxqdmy0sfk-rustc-1.73.0"

Used rustc from the following path: "/nix/store/gz7llp706g0apmsiswwisbpxqdmy0sfk-rustc-1.73.0/bin/rustc"
It looks like Rustup is not being used. For non-Rustup setups, the wasm32-unknown-unknown target needs to be installed manually. See https://rustwasm.github.io/wasm-pack/book/prerequisites/non-rustup-setups.html on how to do this.

gzip: pkg/lldap_app_bg.wasm: No such file or directory
```

### Additional context

This seems to be a general error for wasm builds using Rust 1.73.0. I tried building [pagefind](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/misc/pagefind/default.nix#L104) from unstable as well, which produced the same error.

### Notify maintainers

@emilylange
@bendlas

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
this path will be fetched (0.00 MiB download, 0.00 MiB unpacked):
  /nix/store/8ckiaa7f3lzzb6mqvr32kavbmwf8s8fg-nix-info
copying path '/nix/store/8ckiaa7f3lzzb6mqvr32kavbmwf8s8fg-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.10, NixOS, 23.11 (Tapir), 23.11.20231117.c757e9b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos-22.11"`
 - nixpkgs: `/nix/store/3s69yxbbl116zwga3i6cy7prplywq0bn-source`
```

**Comment by emilylange** [comment]
Hi, thanks for reporting.

Hydra build log: https://hydra.nixos.org/build/241390594

Fix is already in #268168, but hasn't been merged to `master` yet.

Expect it to be resolved soon :)

**Comment by Scandiravian** [comment]
That's great to hear. Thanks for the quick reply and, in general, for maintaining the lldap package; I'm very appreciative of that.
